activities/communication/sign-up.md

```diff
@@ -0,0 +1,18 @@
+---
+Type: guide
+Explains: how to manage sign ups for the newsletter and community calls
+---
+
+# Managing newsletter and community call sign up
+
+We use [Odoo](../tool-management/odoo.md) to manage the sign up process. The modules we use are Surveys and Contacts. In Odoo we have a survey called *Newsletter and community calls*, which configures the [sign up form](https://odoo.publiccode.net/survey/start/594b9243-c7e5-4bc1-8714-35137c971842).
+
+The link for this form can be shared and used wherever we feel that those we are communicating with may be looking for ways to stay in touch with us.
+
+## Update contacts
+
+When preparing to send out [invitations to community calls](../standard-maintenance/preparing-community-call.md) or the newsletter, first the contacts need to be updated. Check in the notes in the bottom of the survey when it was last updated. In the top right corner of the survey there is a button stating how many answers the survey has. Clicking it reveals all the persons that have signed up and their options. By selecting all the answers that were `Created on` later than when it was last updated and then the download button in the top left corner you get all the answers in one file. With the details from the file, update existing contacts or create new ones. Use the tags ´Newsletter´, ´Community Call / Foundation', ´Community Call /Standard´ and ´SendValuable´ to store their preferences for being contacted.
+
+## Log the update
+
+After the contacts have been updated, use the function **Log note** in the bottom of the survey to leave a message that contacts have been updated (a timestamp will be added automatically).
```

Comment:
```suggestion
After the contacts have been updated, use the function **Log note** in the bottom of the survey to leave a message that contacts have been updated (a timestamp will be added automatically).

Delete the contacts file you downloaded, so that we can be confident we're GDPR compliant.
```